code/lib/blocks/src/components/ArgsTable/ArgsTable.stories.tsx

```diff
@@ -61,7 +60,6 @@ export const InAddonPanel = {
 export const InAddonPanelWithWarning = {
```

Comment:
This story name suggests a warning should be there, if we're removing it altogether we should remove the story as well

Reply:
Good catch on both of these. Removed!